Formula/f/fuego.rb

```diff
@@ -2,10 +2,18 @@ class Fuego < Formula
   desc "Collection of C++ libraries for the game of Go"
   homepage "https://fuego.sourceforge.net/"
   url "https://svn.code.sf.net/p/fuego/code/trunk", revision: "1981"
-  version "1.1.SVN"
-  revision 7
+  version "1.1"
+  license any_of: ["GPL-3.0-only", "LGPL-3.0-only"]
+  revision 8
   head "https://svn.code.sf.net/p/fuego/code/trunk"
 
+  # Various other files with differing versions are present in
+  # the SourceForge repository, so we can't rely on the SourceForge strategy.
+  livecheck do
+    url "https://sourceforge.net/p/fuego/activity/feed"
```

Comment:
```suggestion
  livecheck do
    url "https://sourceforge.net/projects/fuego/rss?path=/fuego"
```

The comment seems to suggest that livecheck isn't using the `Sourceforge` strategy here but this check ends up applying the `Sourceforge` strategy to the `livecheck` block URL and checks `https://sourceforge.net/projects/fuego/rss`.

If you intended to check the URL in the `livecheck` block instead, this will need to use `strategy :page_match` or `strategy :xml` and reworked a bit (i.e., the activity feed contains revision numbers but the formula `version` doesn't reference the revision at all, so returning revision numbers would be an issue for version comparison).

That said, it's not usual to need to provide a more specific regex to the `Sourceforge` strategy, so we can omit the comment here.

Reply:
We need to regex the SourceForge RSS feed in order to grab 1.1 instead of the 1.1.4 which is currently mistakenly returned. 1.1.14 is the version of something else in an "old files" directory. Any help to do this the right way would be appreciated!

Reply:
If we only want to match the tarball versions (e.g., `1.1`), then I'm good with this regex (i.e., we only need to change the `url`, so it reflects what it's actually checking). However, the formula isn't using the stable 1.1 version from the tarball, so using `version "1.1"` in the formula would be misleading.

Since the formula is using a revision, we may be better off using the revision number as the formula version (i.e., `version "1981"`). However, the downside is that we wouldn't know when a new stable tarball is available (if ever). A `livecheck` block for this setup would need to check https://sourceforge.net/p/fuego/activity/feed (using either `strategy :page_match` or `strategy :xml`) and a regex that matches the revision numbers from the URLs in the feed.

The middle ground between those two approaches would be to include both the most recent stable version and the revision in the version, maybe like `1.1-1981`. The `livecheck` block would need to check the main SourceForge RSS feed first (https://sourceforge.net/projects/fuego/rss?path=/fuego) and then check the activity feed (https://sourceforge.net/p/fuego/activity/feed) in a `strategy` block. I would include some conditional logic in the `strategy` block to skip fetching the revision information if the main version is newer than 1.1.

Looking at how other package managers handle `fuego`, they use `1.1`, the revision (`2033`), or just `HEAD`. I think using a version that includes both the most recent stable a revision may be best for now, as it would tell us when a new revision is available (`1981` is from 2015-01) and also when a new stable tarball is available (so we can switch to that instead of unstable revisions).

Thoughts on this? Once we decide how to proceed in terms of the version, I'll add a suggestion for the `livecheck` block.

Reply:
I do not have any strong feelings on how we handle it. The project doesn't appear to be active anymore. Just trying to fix current state so it stops saying there is an update: `fuego: 1.1.SVN ==> 1.1.4`

Reply:
This approach seems fine to me @p-linnane.

Reply:
There are newer revisions (the most recent, 2038, is from 2019-12-10) but yeah, it doesn't seem like they're going to release a new stable version anytime soon. If we're not interested in updating to newer revisions and only care about new stable releases, then `version "1.1"` is fine enough.

For what it's worth, I just realized that I had a typo above that flipped my meaning. I meant to say, "it's not unusual to need to provide a more specific regex to the `Sourceforge` strategy" but ended up typing "it's not usual".

I'm good with this if we merge this suggestion to update the `livecheck` block `url` (to what it's really checking) and remove the misleading comment.